recipes/ryvencore-qt/setup.patch

```diff
@@ -0,0 +1,21 @@
+From f9051dbd631814ac8b44e3b3638c5ce04f22d4fa Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Mon, 17 Jan 2022 11:44:47 -0700
+Subject: [PATCH] Update setup.cfg
+
+---
+ setup.cfg | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.cfg b/setup.cfg
+index 064ee25..7c6af9e 100644
+--- a/setup.cfg
++++ b/setup.cfg
+@@ -21,7 +21,6 @@ include_package_data = True
+ python_requires = >=3.6
+ install_requires = 
+ 	ryvencore ==0.3.1.*
+-	PySide2
```

Comment:
As soon as https://github.com/conda-forge/pyside2-feedstock/pull/92 is merged this patch can be removed.